core/input/input_event.h

```diff
@@ -55,6 +55,10 @@ enum ButtonList {
 	BUTTON_MASK_LEFT = (1 << (BUTTON_LEFT - 1)),
 	BUTTON_MASK_RIGHT = (1 << (BUTTON_RIGHT - 1)),
 	BUTTON_MASK_MIDDLE = (1 << (BUTTON_MIDDLE - 1)),
+	BUTTON_MASK_UP = (1 << (BUTTON_WHEEL_UP - 1)),
+	BUTTON_MASK_DOWN = (1 << (BUTTON_WHEEL_DOWN - 1)),
```

Comment:
Typo: next commit changes those to BUTTON_MASK_WHEEL_UP and BUTTON_MASK_WHEEL_DOWN